gen/definitions/classes.yaml

```diff
@@ -191,6 +191,37 @@ fvFBRoute:
   resource_notes:
     - "This resource should not be used in combination with the `vrf_fallback_routes` nested attribute of the `aci_vrf_fallback_route_group` resource. Doing so will result in unexpected behaviour."
 
+fvBD:
+  resource_name: "bridge_domain"
+
+l3extInstP:
+  resource_name: "external_network_instance_profile"
+
+fvSiteAssociated:
+  include: true
+  resource_name: "associated_site"
+  ui_locations:
+    - "Not shown in UI"
+  sub_category: "Multi-Site"
+  dn_formats:
+  - "uni/tn-{name}/ctx-{name}/stAsc"
+  - "uni/tn-{name}/ap-{name}/epg-{name}/stAsc"
+  - "uni/tn-{name}/BD-{name}/stAsc"
+  - "uni/tn-{name}/out-{name}/instP-{name}/stAsc"
+  - "uni/tn-{name}/mscGraphXlateCont/epgDefXlate-[{epgDefDn}]/stAsc"
+
+fvRemoteId:
+  ui_locations:
+    - "Not shown in UI"
+  sub_category: "Multi-Site"
+  dn_formats:
+  - "uni/tn-{name}/ctx-{name}/stAsc/site-{siteId}"
+  - "uni/tn-{name}/ap-{name}/epg-{name}/stAsc/site-{siteId}"
+  - "uni/tn-{name}/BD-{name}/stAsc/site-{siteId}"
+  - "uni/tn-{name}/out-{name}/instP-{name}/stAsc/site-{siteId}"
+  - "uni/tn-{name}/ap-{name}/esg-{name}/stAsc/site-{siteId}"
+  - "uni/tn-{name}/mscGraphXlateCont/epgDefXlate-[{epgDefDn}]/stAsc/site-{siteId}"
```

Comment:
to verify this was needed to manipulate which dn_format was used? what happens when not provided for this class?

Reply:
When not provided, the dn formats would not show up in the md files and the import would also use a non-optimal dn.

Reply:
But your classes.yaml suggestion fixed it. Thanks